patches/react-native-text-input-mask+0.9.0.patch

```diff
@@ -9,7 +51,33 @@ patch-package
  export { mask, unmask, setMask }
  
  export default class TextInputMask extends Component {
-@@ -68,4 +69,13 @@ export default class TextInputMask extends Component {
+@@ -30,7 +31,11 @@ export default class TextInputMask extends Component {
+ 
+     if (this.props.mask && !this.masked) {
+       this.masked = true
+-      setMask(findNodeHandle(this.input), this.props.mask)
++      if(Platform.OS === "android") {
++        setMask(findNodeHandle(this.input), this.props.mask, this.props.isNumericSecureKeyboard === true)
```

Comment:
Could this if/else changed with this:

```typescript
setMask(findNodeHandle(this.input), this.props.mask, Platform.select({
      ios: false,
      android: this.props.isNumericSecureKeyboard === true
    }))
```

Reply:
@Undermaken in this case it is better to leave the if-else statement because on ios the setMask method accepts only 2 parameters